src/styles/compliance.scss

```diff
@@ -0,0 +1,33 @@
+.ob-compliance {
```

Comment:
ideally none of these are nested under this class. So that devs can override styles easier. Obviously these ones need to stay nested: `&.odd > .ob-compliance__container`